### PR TITLE
Docs: Corrected links and anchors

### PR DESCRIPTION
**Pull request by jobr97**
# Description

All relative links in the documentation were missing `'/hassbian-scripts'` and therefore resulted in 404. This PR fixes that along with some anchors (mainly in `/cli/helpers`).

[guidelines]: https://github.com/home-assistant/hassbian-scripts/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jobr97, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassbian-scripts#299).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/contribute/index.md [hunk 1]
```diff
@@ -2,11 +2,15 @@ _Scripts used in the Hassbian image._
...
 ***
 
-[HOME](/) | [SUITES](/suites) | [HASSBIAN-CONFIG (CLI)](/cli) | [**CONTRIBUTE**](/contribute)
+[HOME](/hassbian-scripts/) | [SUITES](/hassbian-scripts/suites) | [HASSBIAN-CONFIG (CLI)](/hassbian-scripts/cli) | [**CONTRIBUTE**](/hassbian-scripts/contribute)
 
 ***
 
-[SUITES](/contribute/suites) | [FUNCTIONS](/contribute/functions)
+[SUITES](/hassbian-scripts/contribute/suites) | [FUNCTIONS](/hassbian-scripts/contribute/functions)
+<<<<<<< HEAD
```
Comment (Landrash) [inline docs/contribute/index.md]: Merge diff left in commit

**Review by Landrash — CHANGES_REQUESTED** [review]
Looks good with one minor tweak necessary

**Comment by jobr97** [comment]
Should be good now @Landrash 😄

File: docs/cli/index.md [hunk 2]
```diff
@@ -2,10 +2,14 @@ _Scripts used in the Hassbian image._
...
+[HOME](/hassbian-scripts/) | [SUITES](/hassbian-scripts/suites) | [**HASSBIAN-CONFIG (CLI)**](/hassbian-scripts/cli) | [CONTRIBUTE](/hassbian-scripts/contribute)
 
 ***
 
-[CLI()](/cli/cli) | [HELPER FUNCTIONS](/cli/helpers)
+<<<<<<< HEAD
+[CLI](/hassbian-scripts/cli/cli) | [HELPER FUNCTIONS](/hassbian-scripts/cli/helpers)
+=======
+[CLI()](/hassbian-scripts/cli/cli) | [HELPER FUNCTIONS](/hassbian-scripts/cli/helpers)
+>>>>>>> f989be3d831fb22831166612aca6cf108fd2a1cd
```
Comment (ludeeus) [inline docs/cli/index.md]: One more :)

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: docs/cli/index.md (hunk 2 above)
Comment (jobr97) [reply docs/cli/index.md]: Damn... 
This should be the last one 😅

**Review by Landrash — APPROVED** [review]

**Comment by Landrash** [comment]
@Luddeus feel free to merge when your block is gone 👍

**Review by ludeeus — APPROVED** [review]

**Pull request merged**